CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 4.10.0
+## Bugfixes
```

Comment:
We use `###` so it's indented inside every version. Can't wait for the automatic version to do that though :)